### PR TITLE
v5.30.3

### PR DESCRIPTION
**Pull request by njalerikson**
Updated version and SHAs. Reset build #. Fixed license to use correct SPDX Identifier.

Checklist
* [X] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

Since v5.32.0 is not available for windows (see #40) let's try to build v5.30.3 first.

### CODE REVIEW

**Comment by njalerikson** [comment]
@conda-forge-admin, please rerender

**Comment by njalerikson** [comment]
@jakirkham @mingwandroid @msarahan - what is needed here to continue?

**Comment by scopatz** [comment]
That was for a previous compiler migration. Please reset the build number to zero since the version itself changed

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,8 +32,7 @@ test:
 
 about:
   home: http://www.perl.org/
-  license: Perl Artistic
-  license_family: Other
+  license: Artistic-1.0-Perl
```
Comment (jakirkham) [inline recipe/meta.yaml:35]: Can we list the `license_file` here as well?

Comment (njalerikson) [reply recipe/meta.yaml:35]: Sure, I didn't include it as I assumed it was defaulting to LICENSE

Comment (njalerikson) [reply recipe/meta.yaml:35]: Ah, I think I understand now, this keyword specifies that the license is injected into the conda package, it has nothing to do with the LICENSE.txt in this feedstock, is this correct?

Comment (scopatz) [reply recipe/meta.yaml:35]: that's right

Comment (jakirkham) [reply recipe/meta.yaml:35]: We might need to do a different thing for Windows as this repackages [Strawberry Perl]( http://strawberryperl.com ) there. Could use a selector to separate how the UNIX and Windows cases are handled.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -32,8 +32,8 @@ test:
...
+  license_file: LICENSE
```
Comment (jakirkham) [inline recipe/meta.yaml:36]:
```suggestion
  license_file:
    - Artistic  # [unix]
    - Copying   # [unix]
    - licenses  # [win]
```

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml:35]: This should do the trick ( https://github.com/conda-forge/perl-feedstock/pull/42/files#r478641858 )

File: recipe/meta.yaml (hunk 2 above)
Comment (njalerikson) [reply recipe/meta.yaml:36]: You're right, I was just about to ask about how to deal with the windows licenses

Comment (jakirkham) [reply recipe/meta.yaml]: Yeah it should grab the whole directory in this case. If not, we can dig into it a bit.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -32,8 +32,11 @@ test:
 
 about:
   home: http://www.perl.org/
-  license: Perl Artistic
-  license_family: Other
+  license: Artistic-1.0-Perl
+  license_file:
+    - Artistic  # [unix]
+    - Copying   # [unix]
+    - licenses  # [win]
```
Comment (jakirkham) [inline recipe/meta.yaml:39]: Looks like [the job is failing]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=201721&view=logs&jobId=a70f640f-cc53-5cd3-6cdc-236a1aa90802&j=a70f640f-cc53-5cd3-6cdc-236a1aa90802&t=f5d15007-a01c-5ad8-c9ce-4d519d3b275f ). Here's [the raw log attached]( https://github.com/conda-forge/perl-feedstock/files/5138461/perl_win_64_20200827.3_log.txt ). Maybe we can tweak this like so?




```suggestion
    - licenses/perl/*  # [win]
```

Comment (njalerikson) [reply recipe/meta.yaml:39]: I sure wouldn't know any better, searching through other conda-forge recipes to see if any others have a complex license_file setup - no luck yet

Comment (jakirkham) [reply recipe/meta.yaml]: The latest change seems sensible. Was pretty sure directories would be handled correctly, but it looks like I'm wrong.

**Comment by jakirkham** [comment]
Thanks @njalerikson! 😄 Will wait to see if @scopatz has any more thoughts 🙂

**Comment by scopatz** [comment]
No more thoughts! This LGTM!

**Pull request merged**